nrf_security/src/psa/psa_eits_zephyr_settings.c

```diff
@@ -10,13 +10,49 @@
 #include <sys/printk.h>
 #include <storage/flash_map.h>
 
+#include <nrf_cc3xx_platform.h>
+#include <nrf_cc3xx_platform_ctr_drbg.h>
+#include <nrf_cc3xx_platform_derived_key.h>
+
 #include "psa_eits_backend.h"
 
 LOG_MODULE_REGISTER(psa_eits_zephyr_settings,
 		    CONFIG_PSA_EITS_BACKEND_ZEPHYR_LOG_LEVEL);
 
+
+const uint8_t deriv_context[]="static_deriv_context";
```

Comment:
I think there are several cases of formatting issues.
Perhaps try running the PR with this clang format file: https://github.com/nrfconnect/sdk-nrf/blob/master/.clang-format